### ISSUE TITLE
why initialize running_var is  0.8999

### ISSUE THREAD

**Issue by CarryHJR** [ultralytics/ultralytics#17469]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question
```
def check_bn_init(model):
    for name, m in model.named_modules():
        if isinstance(m, nn.BatchNorm2d):
            print(f"{name} running_var initial value: {m.running_var[0].item()}")
            break

scale = 'l'
from ultralytics import YOLO
model1 = YOLO(f'yolo11{scale}.yaml', verbose=False).model
check_bn_init(model1)


# model.0.bn running_var initial value: 0.8999999761581421
```

**Comment by UltralyticsAssistant** [comment]
👋 Hello @CarryHJR, thank you for reaching out to us with your query about the initialization of `running_var` 🚀! This is an automated response to guide you, and an Ultralytics engineer will assist you soon.

For a deep dive into the setup and initialization, I'd recommend checking our [Docs](https://docs.ultralytics.com) which have a wealth of information. If this is related to a specific behavior or a 🐛 bug you've encountered, a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) would be invaluable to help us debug the issue effectively.

If this is more of a theoretical inquiry or ❓ question about model behavior/rendering, please provide further context or any related logs you might have.

To ensure you're using the latest features and fixes, please make sure your `ultralytics` package is up-to-date. You can update it by running:

```bash
pip install -U ultralytics
```

### Environments

YOLO models can be run in any of the following environments, which come with the necessary dependencies and tools preinstalled:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>

For real-time support, you can also connect with our community on [Discord](https://ultralytics.com/discord) 🎧. We welcome in-depth discussions and knowledge sharing on [Discourse](https://community.ultralytics.com) or our [Subreddit](https://reddit.com/r/ultralytics).

### Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, it means our continuous integration tests are passing, ensuring that YOLO operations across different systems and conditions are performing as expected.

Thank you for your patience and contributions to our community!

**Comment by Y-T-G** [comment]
Because there's a forward pass whilst creating it.

https://github.com/ultralytics/ultralytics/blob/2c6cd6814402872fa1a08521a2bb8fd8471830ec/ultralytics/nn/tasks.py#L324

**Comment by CarryHJR** [comment]
could be more clear? i cannot found 0.8999999761581421 in the repository

**Comment by Y-T-G** [comment]
@Laughing-q Should the model be in `eval()` mode so that BatchNorm stats don't change due to the forward pass here?
